autosklearn/evaluation/train_evaluator.py

```diff
@@ -400,6 +414,8 @@ def fit_predict_and_loss(self, iterative: bool = False) -> None:
                             if hasattr(self.Y_train, "iloc")
                             else self.Y_train[train_indices],
                             train_pred,
+                            # TODO: Check whether this is the correct xdata
```

Comment:
How do we check this TODO?

Reply:
Sorry, I accidentally left this in the code. Removed.